RELEASE_NOTES.md

```diff
@@ -75,6 +75,11 @@ This will then:
 2. enable the generation and embedding of a maven descriptor (optional if you fully manage your pom.xml with all dependencies)
 3. map P2 dependencies to maven dependencies (optional, but most likely required to get good results)
 
+### Default value change for trimStackTrace
+
+tycho-surefire-plugin had set the default value of the trimStackTrace option to true.
+The default will now be aligned with maven-surefire-plugin at false and will need to be manually adjusted for users that really need the stack traces trimmed.
+
```

Comment:
Can you add a small snippet that shows how to restore old behavior?

Reply:
sure :)